docs/freebsd/README.md

```diff
@@ -2,7 +2,36 @@
 
 ## How to run syzkaller on FreeBSD using qemu
 
-So far the process is tested only on linux/amd64 host. To build Go binaries do:
+So far the process is only tested am amd64 based hosts.
```

Comment:
s/am/on/

Reply:
Fixed in https://github.com/google/syzkaller/pull/1025/commits/d0c59b46bf62cb1f9cd262d2d73fc40e1b70fc0d.